docs/modules/ROOT/pages/index.adoc

```diff
@@ -6,4 +6,18 @@ Welcome to the {product} for Users documentation site.
 
 Use the search field at the top of this page, navigate through topics with the tree on the left or see the links below.
 
-TIP: Contributions to the documentation are welcome. Just click on btn:[Edit this Page] top-right or head over to https://github.com/appuio/appuio-cloud-docs[GitHub] where the content source is stored.
+[TIP]
+====
+Contributions to the documentation are welcome.
+Just click on btn:[Edit this Page] top-right or head over to https://github.com/appuio/appuio-cloud-docs[GitHub] where the content source is stored.
+====
+
+== Disclaimer
+
+{product} is based on https://docs.openshift.com/container-platform/4.9/welcome/index.html[Red Hat OpenShift 4.9], and contains many features not available in other Kubernetes-based platforms.
```

Comment:
```suggestion
{product} is based on https://docs.openshift.com/container-platform/[Red Hat OpenShift], and contains many features not available in other Kubernetes-based platforms.
```

We should not sprinkle versions on too many places, as we will certainly miss them when doing upgrades. The exact version is documented in the zone listing.